### PR TITLE
make sure protocol is set for viteServerUri

### PR DESCRIPTION
**Pull request by franzkugelmann**
[BUGFIX] make sure protocol is set for viteServerUri

### CODE REVIEW

**Comment by franzkugelmann** [comment]
we experienced some cases where the browser blocked uris starting with // instead of https://

File: Classes/EventListener/MutateContentSecurityPolicy.php [hunk 1]
```diff
@@ -26,7 +26,7 @@ public function __invoke(PolicyMutatedEvent $event): void
         $request = $event->request ?? $GLOBALS['TYPO3_REQUEST'] ?? new ServerRequest();
         $viteServerUri = $this->viteService->determineDevServer($request);
         $uris = [
-            new UriValue((string)$viteServerUri),
+            new UriValue('https://' . $viteServerUri->getHost() . ':' . $viteServerUri->getPort()),
```
Comment (s2b) [inline Classes/EventListener/MutateContentSecurityPolicy.php:29]: Hm, I wouldn't want to hard-code the protocol here. Maybe we could define it as a fallback? Or even better: Add both http and https?

Comment (franzkugelmann) [reply Classes/EventListener/MutateContentSecurityPolicy.php:30]: Yepp, should be fine to set both.

Comment (franzkugelmann) [reply Classes/EventListener/MutateContentSecurityPolicy.php:30]: Pull request adapted.

**Comment by s2b** [comment]
CGL errors are now fixed in main. Can you rebase?

**Pull request merged**